src/flexalloc_mm.c

```diff
@@ -1161,7 +1162,7 @@ fla_object_slba(struct flexalloc const * fs, struct fla_object const * obj,
 {
   const struct fla_pool_entry * pool_entry = &fs->pools.entries[pool_handle->ndx];
   return fla_geo_slab_lb_off(fs, obj->slab_id)
-    + (pool_entry->obj_nlb * obj->entry_ndx);
+         + (pool_entry->obj_nlb * obj->entry_ndx);
```

Comment:
Minor nit, whitespace change. Be sure to run the style script in case you haven't.